bliss._.js

```diff
@@ -58,10 +58,12 @@ Object.defineProperty(Array.prototype, _, {
 if (self.EventTarget && "addEventListener" in EventTarget.prototype) {
 	var addEventListener = EventTarget.prototype.addEventListener,
 	    removeEventListener = EventTarget.prototype.removeEventListener,
-	    equal = function(callback, capture, l){
-	    	return l.callback === callback && l.capture == capture;
+	    equal = function(callback, capture, l) {
+			return l.callback === callback && l.capture == capture;
```

Comment:
I will have a look at this later today.
I was using `eslint --fix` so there may be some cases where it is a bit aggressive.